clang/docs/LanguageExtensions.rst

```diff
@@ -2438,6 +2438,17 @@ their usual pattern without any special treatment.
   // Computes a unique stable name for the given type.
   constexpr const char * __builtin_sycl_unique_stable_name( type-id );
 
+``__builtin_sycl_unique_stable_id``
+----------------------------------
+
+Like ``__builtin_sycl_unique_stable_name``, this builtin generates a unique and
```

Comment:
I guess that due to an added prefix, names generated by this built-in are not always demanglable contrary to `__builtin_sycl_unique_stable_name`. Probably should be noted in docs somewhere

Reply:
Thats a good idea!